### PR TITLE
Make iota-wiki compatible ;)

### PR DESCRIPTION
**Pull request by Dr-Electron**
Hi,

I started to add some stuff to make your tool compatible with our wiki build action. Tell me if you don't like my solution 😅

### CODE REVIEW

**Comment by Dr-Electron** [comment]
I thought about splitting the replace and yarn run stuff. But not sure if that wouldn't just add useless stuff. What do you think?

File: src/commands/start.ts [hunk 1]
```diff
@@ -18,24 +18,27 @@ export default class Start extends Command {
   async run() {
     const PWD = process.env.PWD ?? ''
     const userConfig = await getLocalConfig()
-    const WIKI_GIT_FOLDER = join(PWD, userConfig.localWikiFolder, 'iota-wiki')
+    const WIKI_GIT_FOLDER = join(PWD, userConfig.localWikiFolder ?? '', userConfig.localWikiFolder ? 'iota-wiki' : '')
```
Comment (eike-hass) [inline src/commands/start.ts:21]: maybe we should rename `WIKI_GIT_FOLDER` to `WORKING_FOLDER` or something like that.

**Comment by eike-hass** [comment]
> I thought about splitting the replace and yarn run stuff. But not sure if that wouldn't just add useless stuff. What do you think?

I think we can clean that up later, let's merge it with the renaming and see it improves the wiki build.

File: src/commands/start.ts (hunk 1 above)
Comment (Dr-Electron) [reply src/commands/start.ts:21]: Sounds good. But then I will also rename `localWikiFolder` to `workingFolder`

Comment (eike-hass) [reply src/commands/start.ts:21]: nah, would leave that. the config actually specifies the the local wiki folder

**Pull request merged**